### DISCUSSION TITLE
I'm new to this field and I have a question.

### DISCUSSION THREAD

**Discussion by cookieleeuns** [NationalSecurityAgency/ghidra#6731] [Q&A] [answered]
hello, I am very new to this field so my question may be rude. Sorry in advance. I'm frustrated so I came here to ask for advice.
I was using Ghidra to translate the strings used in the program into the desired language, but even if I export the program as "Raw Bytes" or "Original File", it keeps being saved as the original, unmodified exe file. Even if I modified the metadata of the program, it came out as is, and I was annoyed because it kept coming back as the original text. Maybe I'm doing something wrong?

**Comment by ryanmkurtz** [comment]
The "Original File" exporter will only export modified bytes that can be directly mapped back to the file.  Hover over the address in the Listing that you are trying to modify.  Does the hover popup show a "Byte Source Offset" field?

**Comment by dev747368** [comment, marked as answer]
For the string translation feature, there is no change to the bytes in the program.  This feature is strictly a visual feature for the user to be able to change what the string looks like in the listing and the decompiler.

To make this exportable, you would need to modify the bytes at the string's location, typically by using the Byte Viewer tool.  You will have all the normal issues with fitting your new bytes into the same footprint as the old string.
